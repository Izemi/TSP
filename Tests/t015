#!/bin/bash
# Whitespace in city names

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


$BIN/run -stderr=/dev/null ./TSP $WHERE/t015.in -given 'Dar es Salaam' 'Newcastle upon Tyne' 'San Francisco' 'San Pedro Sula' 'Hong Kong' 'New Haven' < /dev/null
