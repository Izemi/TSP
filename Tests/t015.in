Dar es Salaam,-6.87805556,39.2025
Newcastle upon Tyne,55.03805556,-1.68972222
San Francisco,37.61888889,-122.375
San Pedro Sula,15.45194444,-87.92416667
Hong Kong,22.30888889,113.9147222
New Haven,41.26388889,-72.88694444
