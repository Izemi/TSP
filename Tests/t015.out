-given    :     60804.22 Dar es Salaam Newcastle upon Tyne San Francisco San Pedro Sula Hong Kong New Haven Dar es Salaam
